allennlp/models/coreference_resolution/coref_v2.py

```diff
@@ -84,12 +85,17 @@ def __init__(self,
                 TimeDistributed(torch.nn.Linear(mention_feedforward.get_output_dim(), 1)))
         self._mention_pruner = SpanPruner(feedforward_scorer)
         self._antecedent_scorer = TimeDistributed(torch.nn.Linear(antecedent_feedforward.get_output_dim(), 1))
-        self._head_scorer = TimeDistributed(torch.nn.Linear(context_layer.get_output_dim(), 1))
+
+        self._endpoint_span_extractor = EndpointSpanExtractor(context_layer.get_output_dim(),
+                                                              combination="x,y",
+                                                              num_width_buckets=max_span_width,
+                                                              span_width_embedding_dim=feature_size,
+                                                              use_bucket_widths=False)
```

Comment:
Sorry, this name confused me - it seems backwards.  I expected `use_width_buckets`, or similar, not `use_bucket_widths`.  Suggestion: change `num_width_buckets` to `num_width_embeddings`, and `use_bucket_widths` to `bucket_widths`.